bioimageio/spec/__main__.py

```diff
@@ -14,19 +14,24 @@
 app = typer.Typer()  # https://typer.tiangolo.com/
 
 
+@app.command()
+def package_model(model_yaml: Path, auto_convert: bool = False):
```

Comment:
General comment: Can we add docstrings to the CLI? I think it would be good to get something meaningful on `package_model --help` etc.
(Fine to do in a separate PR, just wanted to point this out.)

Reply:
totally! 👍